share/spice/github_jobs/github_jobs.js

```diff
@@ -16,24 +18,40 @@
 
         var sourceUrl = 'https://jobs.github.com/positions?description=' + encodeURIComponent(jobs['description']) + "&location=" + encodeURIComponent(jobs['location']);
 
-        Spice.add({
-            id: 'github_jobs',
-            name: 'Jobs',
-            data: api_result,
-            meta: {
-                sourceUrl: sourceUrl,
-                sourceName: 'GitHub',
-                sourceIcon: true,
-                itemType: 'Jobs'
-            },
-            templates: {
-                group: 'base',
-                options: {
-                    content: Spice.github_jobs.content
+        DDG.require('moment.js', function() {
+            Spice.add({
+                id: 'github_jobs',
+                name: 'Jobs',
+                data: api_result,
+                meta: {
+                    sourceUrl: sourceUrl,
+                    sourceName: 'GitHub',
+                    sourceIcon: true,
+                    itemType: 'Jobs'
+                },
+                normalize: function(item) {
+                    return {
+                        title: item.title,
+                        subtitle: item.company,
+                        location: item.location,
+                        description: $('<div/>').html(DDG.strip_html(item.description)).text(),
+                        url: item.url,
+                        created_at: moment(item.created_at).fromNow()
+                    }
                 },
-                detail: false,
-                item_detail: false
-            }
+                templates: {
+                    group: 'text',
+                    options: {
+                        footer: Spice.github_jobs.footer
+                    },
+                    variants: {
+                        tile: 'basic',
```

Comment:
Change this to `basic1` so that the tiles will have the same sizes:

Before:
![screen shot 2015-03-30 at 11 29 17 am](https://cloud.githubusercontent.com/assets/81969/6900413/434ee59c-d6d4-11e4-8006-68db9f7ea25c.png)

After:
![screen shot 2015-03-30 at 11 58 29 am](https://cloud.githubusercontent.com/assets/81969/6900414/4a2a5d7e-d6d4-11e4-9957-85bdb0659cff.png)

If you want it bigger, you could add `tileSnippet: 'large'` in the `variants` object:

![screen shot 2015-03-30 at 12 00 48 pm](https://cloud.githubusercontent.com/assets/81969/6900434/69bd4034-d6d4-11e4-8ea1-522e2bc57b06.png)

Thanks to @sdougbrown for pointing these things out to me.